core/src/main/java/org/stellar/anchor/sep38/Sep38Service.java

```diff
@@ -215,7 +215,7 @@ public GetPriceResponse getPrice(Sep38GetPriceRequest getPriceRequest) throws An
       validateAmountLimit("sell_", sellAmount, sendMinLimit, sendMaxLimit);
     }
 
-    GetRateRequest request =
+    GetRateRequest.GetRateRequestBuilder rrBuilder =
```

Comment:
nit: rename this to `getRateRequestBuilder` since that's what we call it elsewhere.